src/EFCore.Relational/Extensions/Internal/RelationalPropertyInternalExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.EntityFrameworkCore.Extensions.Internal
```

Comment:
```suggestion
namespace Microsoft.EntityFrameworkCore.Metadata.Internal
```

We usually use the namespace of the target type